lib/engine/action/offer.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
```

Comment:
delete this? we don't need an offer, you can just buy the company

Reply:
What do you mean? This is one of the actions in the auction for this game.

Reply:
you don't need to offer something to someone, someone can just buy it instead?

Reply:
This auction has all players having a pile of companies that they are have to offer and they choose which item to auction off and then others bid. I just chose "offer" as the action, but if there is another action I can use I can swap to it. It was the first feature I implemented.